README.md

```diff
@@ -126,25 +223,26 @@ You should see a list of all the files within your working directory.
 
 The command `ls` stands for "**l**i**s**t".
 
-We can use flags on most unix commands to give more specific instructions. Most programs also accept
-flags, or options for execution.
+We can use flags on most unix commands to give more specific instructions. Most
+programs also accept flags, or options for execution.
 
-A flag is denotated by a `-` ("dash"). **Note:** *In some programs, options are passed directly to the
-command and not via flags.*
+A flag is denotated by a `-` ("dash"). **Note:** *In some programs, options are
+passed directly to the command and not via flags.*
 
-A common flag that nearly all programs and commands accept is a standalone `h`, for "**h**elp" or
-"**h**uman".
+A common flag that nearly all programs and commands accept is a standalone `h`,
+for "**h**elp".
 
 ```bash
 $ ls -h
 ```
 
-Single-character options can typically be combined with each other. For example,  in the `ls` command,
-`h` is a suffix on the `l` flag meaning "**h**uman readable formats." They can be combined with `a`
-meaning "**a**ll information including permissions". Try these three together:
+Single-character options can typically be combined with each other. For example,
+in the `ls` command, `h` is a suffix on the `l` flag meaning "**h**uman readable
+formats." They can be combined with `a` meaning "**a**ll information including.
+Try these three together:
```

Comment:
let's break this down more:

try `ls -l`
try `ls -h`
try `ls -lh` (whoa...c-c-c-c-c-combo!)